libs/barista-components/experimental/datepicker/src/timeinput.ts

```diff
@@ -90,6 +92,46 @@ export class DtTimeInput {
   }
   private _minute: number | null = null;
 
+  /** The minimum selectable hour. */
+  @Input()
+  get minHour(): string | number | null {
```

Comment:
Shouldn't these be only `number | null`. Can you explain why they need to be a string?

Reply:
Ah yes, I confused it with the min parameter from a different method. Thanks!